kolibri/plugins/epub_viewer/assets/src/views/EpubRendererIndex.vue

```diff
@@ -118,7 +127,16 @@
         <div
           class="column epubjs-navigation"
         >
+          <PreviousButton
+            v-if="contentIsRtl"
+            v-show="!isAtStart"
+            :color="navigationButtonColor"
+            :isRtl="contentIsRtl"
+            :style="{ backgroundColor }"
+            @goToPreviousPage="goToPreviousPage"
+          />
           <NextButton
+            v-else
```

Comment:
Non-blocking comment:

One subtle issue with this strategy is that it reverses tab order between RTL and LTR, which is generally not supposed to happen because tab order should be independent of text direction.

In the quiz page we work around this for the 'Next/Previous' buttons by dynamically changing `dir`:

https://github.com/learningequality/kolibri/blob/f6318032221d72c9e9e54d4f41285b357299acff/kolibri/plugins/learn/assets/src/views/ExamPage/index.vue#L46

https://github.com/learningequality/kolibri/blob/f6318032221d72c9e9e54d4f41285b357299acff/kolibri/plugins/learn/assets/src/views/ExamPage/index.vue#L271-L279

Another nice effect here is that the more important 'next' button gets focus _before_ the 'previous' button even though it visually appears "second".

| Visual / reading order | Tab focus order |
|--|--|
| Quiz questions, Previous, Next | Quiz questions, Next, Previous |

Reply:
True - would probably be a better fix. There is still some overall breakage of RTL epubs at the moment, so can fix this during that follow up.